remap-gff3/bin/remap-gff3.py

```diff
@@ -1,10 +1,6 @@
 #! /usr/bin/env python
 # Contributed by Li-Mei Chiang <dytk2134 [at] gmail [dot] com> (2018)
```

Comment:
It seems to me that based on previous discussion. We can just remove all these information because all the contribution were recorded in git history. It doesn't block this PR, though.